src/sv-pipeline/05_annotation/scripts/compute_AFs.py

```diff
@@ -29,47 +34,144 @@ def create_pop_dict(popfile):
     return pop_dict
 
 
-def gather_allele_freqs(record, all_samples, males, females, pop_dict, pops, no_combos = False):
+def _is_biallelic(record):
```

Comment:
This method is repeated a few times in this PR, should we move it to a single copy in `svtk.utils`?

Reply:
Another good idea; done.